addons/bsp_importer/bsp_reader.gd

```diff
@@ -935,7 +935,8 @@ func read_bsp(source_file : String) -> Node:
 			printerr("Invalid script path: ", post_import_script_path)
 	#print("Post import nodes: ", post_import_nodes)
 	for node in post_import_nodes:
-		node.post_import(root_node)
+		if (node.has_method("post_import")):
+			node.post_import(root_node)
```

Comment:
The has_method check is done before adding to post_import_nodes, so this shouldn't be necessary.

Reply:
If you had errors, it's likely because the script had the method but was not marked as tool.  There was a check we did elsewhere to give better feedback about that that may be applicable here.

Reply:
Yip, it was definitely that.